cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -332,29 +297,23 @@ def test_service_ansible_playbook_confirm(appliance, soft_assert):
         casecomponent: Ansible
         caseimportance: medium
         initialEstimate: 1/6h
+        tags: ansible_embed
     """
     collection = appliance.collections.catalog_items
     cat_item = collection.instantiate(collection.ANSIBLE_PLAYBOOK, "", "", {})
     view = navigate_to(cat_item, "Add")
     assert view.provisioning.is_displayed
     assert view.retirement.is_displayed
-    soft_assert(view.provisioning.repository.is_displayed)
-    soft_assert(view.provisioning.verbosity.is_displayed)
+    soft_assert(view.provisioning.repository.is_displayed, True)
```

Comment:
What's the purpose of putting `True` here? The second argument to `soft_assert` should be an explanation of the failure.

I think this would be more robust running `soft_assert` against `view.provisioning.repository.wait_displayed`, and actually pass an explanation of the failure here.

Reply:
I have added `True` while debugging retirement locator and forgot to remove.
Now removed.